frontend/src/screens/studyBanner.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Box, Title, Text, Anchor, Group, useMantineTheme } from '@mantine/core';
+import { IconArrowRight } from '@tabler/icons-react';
+import { useNavigate } from 'react-router-dom';
+import { colors } from '@theme';
+import { useSearchStudies, useParticipantStudies } from './learner/studies';
+import { ParticipantStudy } from '@api';
+
+
+const StudyBanner: React.FC = () => {
+    const { filteredStudies } = useSearchStudies();
+    const { studies = [] } = useParticipantStudies() as { studies: ParticipantStudy[] };
+    const navigate = useNavigate();
+    const theme = useMantineTheme();
+
+    const [totalCompletedCount, setTotalCompletedCount] = useState<number>(0);
+    const [badgesEarned, setBadgesEarned] = useState<number>(0);
+    const [totalPointsEarned, setTotalPointsEarned] = useState<number>(0);
+    const [hasCompletedStudies, setHasCompletedStudies] = useState<boolean>(false);
+    const [fiveMinuteStudies, setFiveMinuteStudies] = useState<ParticipantStudy[]>([]);
+
+    useEffect(() => {
+        setTotalCompletedCount(studies.reduce((sum, study) => sum + (study.completedCount || 0), 0));
+        setBadgesEarned(studies.reduce((count, study) => count + (study.learningPath?.completed ? 1 : 0), 0));
+        setTotalPointsEarned(studies.reduce((sum, study) => sum + (study.learningPath?.completed ? (study.totalPoints || 0) : 0), 0));
+        setHasCompletedStudies(filteredStudies.some((study) => study.completedAt !== undefined));
+        setFiveMinuteStudies(filteredStudies.filter((study) => study.stages && study.stages[0]?.durationMinutes === 5));
+    }, [studies, filteredStudies]);
+
+    const startRandomFiveMinuteStudy = useCallback(() => {
+        if (fiveMinuteStudies.length > 0) {
+            const randomIndex = Math.floor(Math.random() * fiveMinuteStudies.length);
+            const randomStudy = fiveMinuteStudies[randomIndex];
+            navigate(`/studies/details/${randomStudy.id}`);
+        }
+    }, [fiveMinuteStudies, navigate]);
+
+    const formatValue = (value: number) => (value < 10 ? `0${value}` : value);
+
+    return (
+        <Group
+            align="flex-start"
+            style={{
```

Comment:
alot of these properties are redundant since you are now using Group, please review the mantine documentation to see how this works. we also don't want to write these media queries directly in line, ideally a responsive layout will make this irrelevant